uiowa/uiowa
# Added presave hook to clear tags list and related content conditionally if those blocks are enabled.

Resolves #5181

This PR invalidates the caches of the view blocks `tags_list` `related_content` upon saving a node only if tags and/or related content are enabled in the site's config.

# How to test

1. Enable the display of tags and/or related content in SiteNow Pages/Articles/Person configuration, 
ex: `/admin/config/sitenow/uiowa-pages`. 
2. Create a new page.
3. Save the page.
4. Edit the page and add a tag in the "Relationships" setting .
5. Save the page again.
6. Note that the newly added tag IS showing on the page, preferably test this by viewing the page in a private/incognito window as well
7. Repeat for articles and people as well
